cedar-lean/Cedar/Thm/Authorization/Authorizer.lean

```diff
@@ -269,16 +269,6 @@ theorem satisfied_implies_resource_scope {policy : Policy} {request : Request} {
         (and_true_implies_right_true h₁))
   exact resource_eval_ok_means_resource_in_uid h₁ h₂
 
-/--
-  A generic lemma that relates List.mapM to List.map. Not in Std AFAICT.
--/
-theorem if_f_produces_pure_then_mapM_f_is_pure_map {α β} [Monad m] [LawfulMonad m] {f : α → β} {list : List α} :
```

Comment:
Moved to Thm/Data/List.lean and renamed `mapM_pure`

Reply:
Conversation moved to https://github.com/cedar-policy/cedar-spec/pull/305#discussion_r1587717976